road_to_fit/src/db/schema/admin.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { integer, sqliteTable, text } from 'drizzle-orm/sqlite-core'
+
+export const adminTable = sqliteTable('admin', {
```

Comment:
You can call this as admin instead of adminTable, Same with all the schema definitions

Reply:
fixed